src/components/ebay-file-preview-card/file-preview-card.stories.ts

```diff
@@ -0,0 +1,260 @@
+                category: "Menu Actions",
+            },
+        },
+        seeMore: {
+            type: "number",
+            control: { type: "number" },
+            description:
+                'Passing a number here will convert the card to a "see more" card',
+        },
+        footerTitle: {
+            type: "string",
+            control: { type: "text" },
+            description:
+                "Title to display beneath the file, usually the filename",
+        },
+        footerSubtitle: {
+            type: "string",
+            control: { type: "text" },
+            description: "Subtitle to display beneath the file title",
+        },
+        "onMenu-action": {
+            action: "on-menu-action",
+            description: "Triggered when an action is selected from the menu. ",
+            table: {
+                category: "Events",
+                defaultValue: {
+                    summary: "name, event /* from ebay-menu-button */",
+                },
+            },
+        },
+        "onSee-more": {
+            action: "on-see-more",
+            description: "Triggered when the see more button is clicked",
+            table: {
+                category: "Events",
+                defaultValue: {
+                    summary: "",
+                },
+            },
+        },
+        onDelete: {
+            action: "on-delete",
+            description: "Triggered when the delete button is clicked",
+            table: {
+                category: "Events",
+                defaultValue: {
+                    summary: "",
+                },
+            },
+        },
+        onCancel: {
+            action: "on-cancel",
+            description: "Triggered when the cancel button is clicked",
+            table: {
+                category: "Events",
+                defaultValue: {
+                    summary: "",
+                },
+            },
+        },
+    },
+};
+
+export const Uploading = Template.bind({});
+Uploading.args = {
+    a11yCancelUploadText: "Cancel upload",
+    deleteText: "Delete",
+    file: {
+        name: "file-name.jpg",
+        type: "image/jpeg",
+    },
+    status: "uploading",
+};
+
+Uploading.parameters = {
+    docs: {
+        source: {
+            code: tagToString("ebay-file-preview-card", Uploading.args),
+        },
+    },
+};
+
+export const Image = Template.bind({});
+Image.args = {
+    a11yCancelUploadText: "Cancel upload",
+    deleteText: "Delete",
+    file: {
+        name: "file-name.jpg",
+        type: "image/jpeg",
+        src: "https://ir.ebaystatic.com/cr/v/c01/skin/docs/tb-real-square-pic.jpg",
+    },
+};
+
+Image.parameters = {
+    docs: {
+        source: {
+            code: tagToString("ebay-file-preview-card", Image.args),
+        },
+    },
+};
+
+export const Video = Template.bind({});
+Video.args = {
+    a11yCancelUploadText: "Cancel upload",
+    deleteText: "Delete",
+    file: {
+        name: "file-name.mov",
```

Comment:
Is this story using our ebay-video-player? It probably should, if not.

Reply:
It doesn't need to. 
The ebay-video is a bit overkill for this usecase
I also think this PR might need to be updated to not show the preview. We had discussed with design that it was going to be a static image only.

Reply:
Why doesn't it need to? Will our end users not use the ebay-video component here? 

Moot point if it's going to be a static image.

Reply:
It will be a static image. 
Not playable

Reply:
I see it's the HTML video tag, but just using the `poster` url only with no controls. This is good.

Reply:
Looks like Skin has the `src` set, not `poster`. So we need to clarify which attr is correct (but could be addressed after release), i.e. do we get an image url or movie url back from the backend on upload?

<img width="1203" alt="Screenshot 2024-10-30 at 2 51 56 PM" src="https://github.com/user-attachments/assets/1081f0fd-d292-4ae0-a1cf-a857d2ed08f8">